community/tutorials.rst

```diff
@@ -46,6 +46,7 @@ Video tutorials
 - `Gwizz <https://www.youtube.com/@Gwizz1027>`_ (2D, GDScript).
 - `Quiver <https://quiver.dev/>`_ (2D, GDScript).
 - `Maker Tech <https://www.youtube.com/watch?v=0mUoRdYe0s4>`_ (2D, GDScript).
+- `FencerDevLog <https://www.youtube.com/@FencerDevLog>` _ (2D, 3D, GDScript, Shaders).
```

Comment:
```suggestion
- `FencerDevLog <https://www.youtube.com/@FencerDevLog>`_ (2D, 3D, GDScript, Shaders).
```

Reply:
You keep ignoring this change and marking it as resolved, why? Are you having problems pushing your changes?

Reply:
Sorry, I am not certain what to do here. Is there a problem with the space? I must have overlooked it.

Reply:
You were instructed twice for this? And you marked it as resolved, so you must have seen it right?

Reply:
If you don't know what to do with a suggestion please ask instead of marking it as resolved 🙂